test/clj/rems/test_browser.clj

```diff
@@ -100,7 +100,7 @@
 
 (use-fixtures :each btu/fixture-driver)
 
-(use-fixtures :once btu/test-dev-or-standalone-fixture create-test-data)
+(use-fixtures :once btu/test-dev-or-standalone-fixture create-test-data btu/accessibility-report-fixture)
```

Comment:
perhaps fixture per line like in other test namespaces?

Reply:
I don't see it as a problem with just three but it's possible.

Reply:
Looks like we currently use both styles.

Reply:
looks to me like test-browser is the only NS that has multiple fixtures on the same line

Reply:
I found `test_template.clj` and `test_db.clj` too.

Reply:
Also most cases have just one and that is on the same line.

Reply:
I think `(use-fixtures :once foobar)` is fine on one line, but I kinda like seeing multiple fixtures on multiple lines

but I won't insist

Reply:
I can fix them all. My only concern with these in general is just that grep becomes harder as they are not on the same line.